# Used for building the graph update over acomms
int16 x
int16 y
int8 z
uint8 qw
uint8 qx
uint8 qy
uint8 qz
uint8 sigma_x
uint8 sigma_y
uint8 sigma_z
uint8 sigma_psi
uint8 rho_xy
uint8 rho_xpsi
uint8 rho_ypsi